samples/CommunityToolkit.Maui.Sample/App.xaml

```diff
@@ -1,12 +1,53 @@
 ﻿<Application xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
              xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:popups="clr-namespace:CommunityToolkit.Maui.Sample.Views.Popups"
              x:Class="CommunityToolkit.Maui.Sample.App">
     <Application.Resources>
         <ResourceDictionary>
             <ResourceDictionary.MergedDictionaries>
                 <ResourceDictionary Source="Resources/Styles/Colors.xaml" />
                 <ResourceDictionary Source="Resources/Styles/Styles.xaml" />
             </ResourceDictionary.MergedDictionaries>
+
+            <Style x:Key="ImplicitPopupStyle" TargetType="{x:Type popups:ImplicitStylePopup}">
```

Comment:
I don't know why I have it in my head the the implicit and explicit options are the other way round. You have to explicitly use this one via `Style="{StaticResource ImplicitPopupStyle}"` if I am correct. Whereas the current `ExplicitStylePopup` is implicitly applied without having to define the `Style` property. Is that correct?

Reply:
You are right. I was naming the styles backwards.
I'll fix it and upload it again.